src/apicast-cli/blank-app/.circleci/config.yml

```diff
@@ -0,0 +1,12 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: quay.io/3scale/s2i-openresty-centos7:1.11.2.3-4
```

Comment:
instead of the image you can use `{{ s2i.builder_image }}` and it will be templated like the others